copyright

```diff
@@ -709,15 +709,15 @@ Comment: Taken from https://www.flickr.com/photos/61135621@N03/17755765778 and c
 
 Files:
  images/land/station28*
```

Comment:
Station15 is used on an Efreti station, and I suppose the new image fits, but station28 is used on an abandoned Korath station. The older image at least has a dark, grimy appearance to it that might be fitting of an abandoned station, but this new one looks far to clean and bright.